src/components/recruit/contstants.ts

```diff
@@ -0,0 +1,117 @@
+export const FAQ_TYPE = {
+  REQUIREMENT: 'REQUIREMENT',
+  INTERVIEW: 'INTERVIEW',
+  ACTIVITY: 'ACTIVITY',
+} as const;
+
+export type FaqType = keyof typeof FAQ_TYPE;
+
+interface FaqItem {
+  title: string;
+  description: string;
+}
+
+interface FaqMap {
+  [key: string]: FaqItem[];
+}
+
```

Comment:
```suggestion
export type FaqType = 'REQUIREMENT' | 'INTERVIEW' | 'ACTIVITY';

interface FaqItem {
  title: string;
  description: string;
}

type FaqMap {
  [key in FaqType]: FaqItem;
}
```

유니온 타입이랑 [맵드 타입](https://www.typescriptlang.org/docs/handbook/2/mapped-types.html) 쓸 수도 있어 보여요! (저는 최근에 배웠는데 유용하더라구요)

그치만 ~ 시간이 시간인지라 그냥 진행해도 문제 없어 보입니다 !!! 👍 👍 👍

Reply:
이것도 적용해보겟어요!